components/playlist/common/playlist_render_frame_observer_helper_unittest.cc

```diff
@@ -131,4 +133,5 @@ INSTANTIATE_TEST_SUITE_P(
                   &UnsupportedSrcSchemeBlobHTTP)),
     [](const auto& info) { return std::get<0>(info.param); });
 
+}  // namespace
```

Comment:
pls don't wrap test classes and test instantiations inside anonymous namespace. Only local functions should be wrapped.

you can't friend anonymous classes.

Reply:
Meant to include the free functions only.